en-us.yml

```diff
@@ -33,4 +76,42 @@ commands:
     - "Maybe not."
     - "Yes!"
     - "No!"
-    - "¯\\_(ツ)_/¯"
\ No newline at end of file
+    - "¯\\_(ツ)_/¯"
+    vemdezap:
+    description: "Good morning catilangaiada! Come from zap baby, use the power of zapzap in your message to show how 🔝 with the Whatsapp's girls!"
+    examples:
+    - "come from zap baby"
+    - "Hi catilangaiada, nobody says nothing kkkkkkkkkkkk"
+    - "zapzap is better than discord! There we have the number phone of girls"
+  actions:
+    examples:
+    - "297153970613387264"
+    - "@MrPowerGamerBR"
+    slap:
+      description: "Use this command for someone that is deserving to receive some slaps"
+      response: "**{0} give a slap in {1}!**"
+      response-anti-idiot: "**How you can sleep in night after try to slap a girl like me? **{0} give a slap in {1}!"
+    kiss:
+      description: "Use this command for kiss someone"
+      response: "**{0} kissed {1}!**"
+      response-anti-idiot: "Nah, I don't want to kiss you... but I still like you like a friend!"
+    attack:
+      description: "Use this command for attack someone"
+      response: "**{0} attacked {1}!**"
+      response-anti-idiot: "**How you dare do this to me? **{0} attacked {1}!"
+    hug:
+      description: "Use this command for give a sweet hug in someone"
+      response: "**{0} hugged {1}!**"
+    dance:
+      description: "Use this command to dance with someone"
+      response: "**{0} danced with {1}!**"
+  # MISCELÂNEA
+  fanarts:
+    description: "Check out the {0} awesome {1} fan arts people made for me!"
+    made-by: "Fan Art made by {0}"
+    thank-you-all: "thank for everybody that made grat and awesome fan arts me!"

```

Comment:
Thanks to everyone that made those awesome fan arts for me!

Reply:
Ainda falta arrumar aqui para que seja "Thanks to everyone that made those awesome fan arts for me!"